Packs/FeedCrowdstrikeFalconIntel/Integrations/CrowdStrikeIndicatorFeed/CrowdStrikeIndicatorFeed.py

```diff
@@ -0,0 +1,407 @@
+        indicator = {}  # type: Dict
+        for resource in raw_response['resources']:
+            indicator = {
+                'type': CROWDSTRIKE_TO_XSOAR_TYPES.get(resource.get('type'), resource.get('type')),
+                'value': resource.get('indicator'),
+                'rawJSON': resource,
+                'fields': {'actor': resource.get('actors'),
+                           'reports': resource.get('reports'),
+                           'malwarefamily': resource.get('malware_families'),
+                           'stixkillchainphases': resource.get('kill_chains'),
+                           'ipaddress': resource.get('ip_address_types'),
+                           'domainname': resource.get('domain_types'),
+                           'targets': resource.get('targets'),
+                           'threattypes': resource.get('threat_types'),
+                           'vulnerabilities': resource.get('vulnerabilities'),
+                           'maliciousconfidence': resource.get('malicious_confidence'),
+                           'updateddate': resource.get('last_updated'),
+                           'creationdate': resource.get('published_date'),
+                           'tags': [label.get('name') for label in resource.get('labels')]  # type: ignore
+                           }
+            }
+            if tlp_color:
+                indicator['fields']['trafficlightprotocol'] = tlp_color
+            parsed_indicators.append(indicator)
+
+        return parsed_indicators
+
+    @staticmethod
+    def build_type_fql(types_list: list) -> str:
+        """ Builds an indicator type query for the filter parameter
+
+        Args:
+            types_list(list): indicator types that was chosen by user
+
+        Returns:
+            fql query (str)
+        """
+
+        if 'ALL' in types_list:
+            # Replaces "ALL" for all types supported on XSOAR.
+            crowdstrike_types = ['username', 'domain', 'email_address', 'hash_md5', 'hash_sha256', 'ip_address',
+                                 'registry', 'url']
+            crowdstrike_types = [f"type:'{type}'" for type in crowdstrike_types]
+
+        else:
+            crowdstrike_types = [f"type:'{XSOAR_TYPES_TO_CROWDSTRIKE.get(type.lower(), type)}'" for type in types_list]
+
+        result = ','.join(crowdstrike_types)
+        return result
+
+    @staticmethod
+    def handle_error_response(res) -> None:
+        """
+        Handle error response and display user specific error message based on status code.
+
+        :param res: response from API.
+        :return: raise DemistoException based on status code.
+        """
+        errors = []
+        try:
+            error_entry = res.json()
+            errors = error_entry['errors']
+        except Exception:  # ignoring json parsing errors
+            pass
+
+        error_message = next((item for item in errors if item['code'] == res.status_code)).get('message')
```

Comment:
So again why `next`?
You could do:
```
error_message = [item.get('message') for item in errors if item['code'] == res.status_code][-1]
```
And why not staying on the safe side and even do something like:
```
error_message = '\n'.join[item.get('message') for item in errors if item['code'] == res.status_code and item.get('message')]
```
This way **if** you will have more than one error it will look like:
error 1
error 2...